Source/WTF/wtf/PlatformJSCOnly.cmake

```diff
@@ -116,7 +116,8 @@ elseif (CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
 else ()
     list(APPEND WTF_SOURCES
         generic/MemoryFootprintGeneric.cpp
-        generic/MemoryPressureHandlerGeneric.cpp
+        morphos/MemoryPressureHandlerMorphOS.cpp
+        # generic/MemoryPressureHandlerGeneric.cpp
```

Comment:
This is the solution for now, for a fix as I described at https://github.com/walkero-gr/webkitty/issues/7#issuecomment-1351388355

When we start building the full WebKit, and not JSCOnly, then this probably will need to be reverted. That's why I left the commented line there.